internal/handlers/jsonb1/where.go

```diff
@@ -34,7 +34,10 @@ func scalar(v any, p *pg.Placeholder) (sql string, args []any, err error) {
 	case string:
 		sql = "to_jsonb(" + p.Next() + "::text)"
 		arg = v
-	case types.ObjectID:
+	case bool:
+		sql = "to_jsonb(" + p.Next() + "::bool)"
+		arg = v
+	case types.ObjectID, float64, int64, *types.Array, types.Binary:
```

Comment:
types.Array is not a scalar type, it should not be there. https://github.com/FerretDB/FerretDB/blob/7466a8091080de4724e9f6b87d923c87a47d39fb/internal/types/types.go#L28-L44

Reply:
Let's make a separate function for now. Once we use it in several places, we will see if it has more in common with `scalar` or not. Right now, I'm not sure if we would need to convert the whole array as often as scalar values.